deploy/helm/quarks-job/values.yaml

```diff
@@ -8,6 +8,8 @@ nameOverride: ""
 
 fullnameOverride: ""
 
+createWatchNamespace: true
```

Comment:
`createWatchedNamespace`? Not sure if it's correct?

Reply:
Not sure about the correct word. But I think createWatchNamespace is fine.